### PR TITLE
#33 Feature: Crud Operation For Buses 

### PR DESCRIPTION
**Pull request by arthurjunior250**
**What does this PR do?**

Implements Crud operations for buses.

**Description of Task to be completed?**

-The operator should be able to create, update, and delete a bus.
-The operator should be able to view a paginated list of buses in the system.
-Users should be able to view the bus's details.
-Skeleton UIs/screen are/is shown when the page loads/is loading.
-The page is responsive.
-The page is responsive.

**How should this be manually tested?**

Clone the repo

checkout to branch `Ft-Crud-Operation-for-Buses-#33`

Run npm install to install modules

Run npm run `dev:start ` to start the development server

-route:`http://localhost:3000/dashboard/Buses`

What are the relevant Pivotal Tracker/trello stories?
link to my pivotal tracker :-https://trello.com/c/gY243nQX

**Screenshots (if appropriate)**
Screenshot from 2022-05-25 17-24-50

N/A
**Questions:**
N/A

### CODE REVIEW

**Review by shemalucien — CHANGES_REQUESTED** [review]
Can you use icons for edit, save, and delete buttons to maintain the consistency of the design? otherwise you did a good job  mn

File: src/tests/landingpage.test.js [hunk 1]
```diff
@@ -83,12 +83,15 @@ const linkElement = screen.getByText("Send");
 // const linkElement = screen.getByText("Contact");
 // });
 
+<<<<<<< HEAD
```
Comment (songa1) [inline src/tests/landingpage.test.js]: Check out this. It might be unfinished merge.

File: src/tests/landingpage.test.js [hunk 2]
```diff
@@ -83,12 +83,15 @@ const linkElement = screen.getByText("Send");
...
+=======
 
 
 
 
 
 
+>>>>>>> develop
```
Comment (songa1) [inline src/tests/landingpage.test.js]: Here too. Check out what's happening!

**Review by songa1 — COMMENTED** [review]
Check out on your merge history and see unfinished one.

**Review by gyssa-prince — CHANGES_REQUESTED** [review]
can you use json db

**Review by Rukundo725 — APPROVED** [review]

**Review by Juleshb — APPROVED** [review]

**Review by Benafrica — APPROVED** [review]

**Review by songa1 — APPROVED** [review]

**Pull request merged**